pages/en/new-design/index.mdx

```diff
@@ -6,12 +6,15 @@ layout: home.hbs
 
 <WithBadge section="index" />
 
+<div className="flex flex-col gap-4">
```

Comment:
Can we avoid adding this div? Reasoning is the less code we add it here, the easier it is for the translators